measure_extinction/plotting/plot_spec.py

```diff
@@ -205,24 +216,33 @@ def plot_multi_spectra(
         (waves, fluxes, flux_uncs) = starobs.get_flat_data_arrays(
             starobs.data.keys() - (exclude + exclude2)
         )
+        if wavenum:
+            x = 1 / waves
+        else:
+            x = waves
         if range is not None:
-            waves = waves[waves >= range[0]]
-        min_wave = waves[0]
+            x = x[x >= range[0]]
```

Comment:
This will, of course, only work if range is also in wavenumbers. The information about "range" in this function (and others) is currently: Wavelength range to be plotted (in micron) - [min,max]. This should probably be changed to something more general.

Reply:
Ah, I see now that you changed it for the plot_spectrum function below, but not for this one.

Reply:
Ok.  Looked into this and am not not sure what needs fixing.

Reply:
Everywhere where it says: "Wavelength range to be plotted (in micron) - [min,max]", you should simply delete the "(in micron)". Then it is up to the user to make sure that they provide the ranges to be plotted in the correct units (either micron or 1/micron)... Or maybe it would be better to mention that, i.e.:
"Wavelength range to be plotted (in micron when plotting wavelengths or in 1/micron when plotting wavenumber) - [min,max]"